### DISCUSSION TITLE
I'm not able to create a vercel Account

### DISCUSSION THREAD

**Discussion by passworx** [vercel/community#2362] [Help] [answered]
### Summary

I'm not able to create a vercel Account.

there is some error "This user account is blocked. Contact registration@vercel.com for more information."

Please help me

### Additional information

![image](https://user-images.githubusercontent.com/132909002/236823841-a5252977-9df5-46de-9e78-3022f4e7d72b.png)

**Comment by amyegan** [comment, marked as answer]
Hi @passworx! If you haven't done it already, please do email [registration@vercel.com](mailto:registration@vercel.com). They can help you figure out this out 🙇
